### PR TITLE
[RDAPI-29487] MFA can be done using a 3rd party identity provider via SSO

### PR DESCRIPTION
**Pull request by dsantosaxway**
## Describe the changes

MFA is not something we can support in API Manager/Gateway, however, we can use SSO and the backing IdP can be configured to use MFA. This needs to be reflected in the documentation.

## Deploy preview link

http://pr-2631.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_administration/apimgr_sso/#identity-provider

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 8fe4128 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2631.axway-open-docs.opendocs-builder.pcloud.axway.int |

**Review by kmercierwhite — APPROVED** [review]

File: content/en/docs/apim_administration/apimgr_sso/_index.md [hunk 1]
```diff
@@ -19,7 +19,15 @@ A Service Provider (SP) protects access to requested resources, such as web site
 
 ### Identity Provider
 
-An Identity Provider (IdP) is a system that creates, maintains, and manages identity information for users, services, or systems, and provides authentication to other service providers (applications) within a network. An IdP is a trusted entity that users and servers can rely on when they are establishing a dialog that must be authenticated. The IdP sends an attribute assertion containing trusted information about the user to the SP. In an Axway deployment, the IdP is a third-party product.
+An Identity Provider (IdP) is a system that creates, maintains, and manages identity information for users, services, or systems, and provides authentication to other service providers (applications) within a network.
+
+An IdP has the following characteristics:
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apimgr_sso/_index.md:24]: There was too much repetition of "An IdP", so I've created a list. Please let me know if my statement is technically correct 
"An IdP has the following characteristics", or do you have a better suggestion?

Comment (kmercierwhite) [reply content/en/docs/apim_administration/apimgr_sso/_index.md:24]: I like this wording! These are all characteristics of an IdP

I would maybe put the sentence "In an Axway deployment, the IdP is a third-party product" into the top paragraph of this change. Just a personal preference to not jump from paragraph, to list, to paragraph.

Comment (andreamussap) [reply content/en/docs/apim_administration/apimgr_sso/_index.md:24]: Yes, even better. I'll make this change. Thanks

**Pull request merged**